assets/components/LogContainer.vue

```diff
@@ -1,14 +1,26 @@
 <template>
   <scrollable-view :scrollable="scrollable" v-if="container">
     <template v-slot:header v-if="showTitle">
-      <div class="mr-0 columns is-vcentered is-hidden-mobile">
-        <div class="column is-clipped">
+      <div class="mr-0 columns is-vcentered is-marginless is-hidden-mobile">
+        <div class="column is-clipped is-paddingless">
           <container-title :container="container" @close="$emit('close')"></container-title>
         </div>
-        <div class="column is-clipped">
+        <div class="column is-clipped is-paddingless">
           <container-stat :stat="container.stat" :state="container.state"></container-stat>
         </div>
-        <div class="column is-narrow">
+        <div class="column is-clipped is-paddingless"></div>
+        <div class="column is-narrow is-paddingless" v-if="closable">
+          <button class="delete is-medium" @click="$emit('close')"></button>
+        </div>
+      </div>
+      <div class="mr-0 is-vcentered is-hidden-mobile">
+        <div class="is-flex">
+          <a @click="onClearClicked" class="button is-small is-outlined" id="clear">
```

Comment:
Since there is only an icon, a tooltip with `title=` could be good.

Reply:
Done. How does this look?

![image](https://user-images.githubusercontent.com/3276350/140439130-71f51322-16a8-40e5-941a-d927fbd62592.png)